go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/aws/aws-sdk-go v1.37.1
 	github.com/channelmeter/iso8601duration v0.0.0-20150204201828-8da3af7a2a61
 	github.com/google/go-cmp v0.3.0 // indirect
+	github.com/google/uuid v1.2.0
 	github.com/kr/pretty v0.2.0 // indirect
```

Comment:
I like to use GUIDs/UUIDs for test data instead of hardcoded values.

Reply:
Yeah, makes sense to me. There are also some libraries to generate fake data, but probably not as useful in this example.